### PR TITLE
implementing issue#15

### PR DESCRIPTION
**Pull request by kruslin2**
Updated testing code

### CODE REVIEW

File: plant.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+        # Initialize the code variables for testing
+        self.transparent = True
+        self.transparency = 2
+        self.is_cube = False
+        self.glass = False
+        self.translucent = False
+        self.colliders = []
+
+        self.vertex_positions = [
```
Comment (cleancoder1016) [inline plant.py:46]: there are no comments to the code after line 45. please add comments to increase to code readability.

File: snow.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        self.tex_coords = [
+	[0.0, 1.0, 0.0,  0.0, 0.0, 0.0,  1.0, 0.0, 0.0,  1.0, 1.0, 0.0],
+	[0.0, 1.0, 0.0,  0.0, 0.0, 0.0,  1.0, 0.0, 0.0,  1.0, 1.0, 0.0],
+]  
+        self.shading_values =  [
+	[1.0, 1.0, 1.0, 1.0],
+	[0.4, 0.4, 0.4, 0.4],
+]
+ 
+
```
Comment (cleancoder1016) [inline snow.py:66]: there are no comments to the code after line 66. please add comments to increase to code readability.

File: util.py [hunk 3]
```diff
@@ -14,15 +17,72 @@
...
+            glm.ivec3(0, 0, 1), 
+            glm.ivec3(0, 0, -1))
+        self.EAST = glm.ivec3(1, 0, 0)
+        self.WEST = glm.ivec3(-1, 0, 0)
+        self.UP = glm.ivec3(0, 1, 0)
+        self.DOWN = glm.ivec3(0, -1, 0)
+        self.SOUTH = glm.ivec3(0, 0, 1)
+        self.NORTH = glm.ivec3(0, 0, -1)
+
+
```
Comment (cleancoder1016) [inline util.py:38]: there are no comments to the code after line 45. please add comments to increase to code readability.

**Review by cleancoder1016 — CHANGES_REQUESTED** [review]

**Comment by cleancoder1016** [comment]
please merge this branch
@kruslin2

File: plant.py (hunk 1 above)
Comment (atanake147) [reply plant.py:46]: Comments added

File: snow.py (hunk 2 above)
Comment (atanake147) [reply snow.py]: Comments added

Comment (kruslin2) [reply snow.py]: Comments were added

File: util.py (hunk 3 above)
Comment (kruslin2) [reply util.py]: Comments were added

**Review by cleancoder1016 — APPROVED** [review]

**Comment by kruslin2** [comment]
[Coverage for door.py_ 100%.pdf](https://github.com/WSUCEG-7140/minecraftclone/files/11992441/Coverage.for.door.py_.100.pdf)
[Coverage for fire.py_ 100%.pdf](https://github.com/WSUCEG-7140/minecraftc
[Coverage for plant.py_ 100%.pdf](https://github.com/WSUCEG-7140/minecraftclone/file
[Coverage for snow.py_ 100%.pdf](https://github.com/WSUCEG-7140/minecraftclone/files/11992456/Coverage.for.snow.py_.100.pdf)
s/11992452/Coverage.for.plant.py_.100.pdf)
lone/files/11992445/Coverage.for.fire.py_.100.pdf)

**Comment by kruslin2** [comment]
[Coverage for door.py_ 100%.pdf](https://github.com/WSUCEG-7140/minecraftclone/files/11992457/Coverage.for.door.py_.100.pdf)
[Coverage for fire.py_ 100%.pdf](https://github.com/WSUCEG-7140/minecraftclone/files/11992458/Coverage.for.fire.py_.100.pdf)

**Comment by kruslin2** [comment]
[Coverage for plant.py_ 100%.pdf](https://github.com/WSUCEG-7140/minecraftclone/files/11992460/Coverage.for.plant.py_.100.pdf)
[Coverage for snow.py_ 100%.pdf](https://github.com/WSUCEG-7140/minecraftclone/files/11992461/Coverage.for.snow.py_.100.pdf)

**Pull request merged**